pkg/antctl/transform/networkpolicy/response.go

```diff
@@ -190,7 +190,7 @@ func (r EvaluationResponse) GetTableRow(_ int) []string {
 		// Handle K8s NPs empty action field. "Allow" corresponds to a K8s NP
 		// allow action, and "Isolate" corresponds to a drop action because of the
 		// default isolation model. Otherwise, display the action field content.
-		action := "Allow"
+		action := ""
```

Comment:
I should have updated the comment as well.
For now, K8s explicit rules will go to the `Action != nil` case with their default allow action, and manual inserted isolation rules will go to `math.MaxInt32`. Based on current implementation, the only case left over is `Action = nil`, (which will never happen as long as Antrea controller is functioning, not for K8s explicit rules), so perhaps defaulting to `""` is more suitable.

Reply:
For the sake of our understanding, could you point to the code that ensures that the action is set to `Allow` K8s NP explicit rules? Do we have a unit test that validates this behavior server-side?

Reply:
The action for K8s rules are set in [controller networkpolicy](https://github.com/antrea-io/antrea/blob/27424c33a0f4c7c3b4d04f2a3b9ca34b40d5f0c1/pkg/controller/networkpolicy/networkpolicy_controller.go#L722), then [controller endpoint querier](https://github.com/antrea-io/antrea/blob/42e7cfaf7399cd56ebde88818b396317d8afd770/pkg/controller/networkpolicy/endpoint_querier.go#L154) and [evaluation](https://github.com/antrea-io/antrea/blob/42e7cfaf7399cd56ebde88818b396317d8afd770/pkg/controller/networkpolicy/endpoint_querier.go#L320). But after discussion, we thought action field should not be set for isolation since it was not an actual rule (being changed in this PR).
It is covered by networkpolicy UT, but not endpoint_querier UT (changing in the PR)